ert/ensemble_evaluator/snapshot.py

```diff
@@ -15,6 +15,7 @@
     get_job_id,
     get_real_id,
     get_step_id,
+    get_job_index,
```

Comment:
(these imports can be sorted alphabetically - or it will be done automatically when we at one point are able to use `isort`)